bin/reset-local-dev

```diff
@@ -61,7 +61,6 @@ source "$GIT_ROOT/bin/setup-kind"
 echo "Deploying Astronomer..."
 helm install -f "$CONFIG_FILE" \
   --namespace "${NAMESPACE}" astronomer \
-  --set global.postgresqlEnabled=true \
```

Comment:
I'm removing because we can just use the cofig files instead of hard coding this here.